From 5ae7c57db92cedd196417dbbda7e756f127bf1ae Mon Sep 17 00:00:00 2001
From: Steeve Morin <steeve@zml.ai>
Date: Wed, 12 Mar 2025 15:05:58 +0000
Subject: [PATCH 8/8] [ROCm] Enable mfma instructions by passing the correct
 arch name

Without this commit, mfma instructions would not be emitted
by this pass.
---
 xla/backends/gpu/codegen/triton/compilation_pipeline_rocm.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xla/backends/gpu/codegen/triton/compilation_pipeline_rocm.cc b/xla/backends/gpu/codegen/triton/compilation_pipeline_rocm.cc
index f6bd98aeda..e45c854ba9 100644
--- a/xla/backends/gpu/codegen/triton/compilation_pipeline_rocm.cc
+++ b/xla/backends/gpu/codegen/triton/compilation_pipeline_rocm.cc
@@ -89,7 +89,7 @@ absl::Status CreateTritonPipeline(
   pm->addPass(mt::gpu::createTritonGPUCoalesce());
   pm->addPass(mt::gpu::createTritonGPURemoveLayoutConversions());
   pm->addPass(mt::gpu::createTritonGPUOptimizeThreadLocality());
-  pm->addPass(mlir::createTritonAMDGPUAccelerateMatmulPass());
+  pm->addPass(mlir::createTritonAMDGPUAccelerateMatmulPass(cc.gfx_version()));
   pm->addPass(mt::gpu::createTritonGPURemoveLayoutConversions());
   // TODO ROCm Check if we want to compare MI100 and greater
   pm->addPass(mlir::createTritonAMDGPUOptimizeEpiloguePass());
-- 
2.43.0

